NumLib/TimeStepping/Algorithms/FixedTimeStepping.h

```diff
@@ -58,6 +60,13 @@ class FixedTimeStepping : public ITimeStepAlgorithm
      */
     FixedTimeStepping(double t_initial, double t_end, const std::vector<double> &vec_all_dt);
 
+    /**
+     * @brief Create timestepper from the given configuration
+     *
+     * Currently thisd function only covers uniform timestep size.
+     */
+    static FixedTimeStepping* newInstance(ConfigTree const& config);
```

Comment:
In GroundwaterFlowProcess there is a line `_A.reset(_global_setup.createMatrix(_local_to_global_index_map->dofSize()));`, i.e. `createMatrix()` returns a plain pointer.
So at least this case is not handled always with `unique_ptr`.
I think, it is not so much of an issue here, because the name `newInstance` already indicates ownership transfer.